asciidoctorj-core/src/main/java/org/asciidoctor/jruby/cli/AsciidoctorInvoker.java

```diff
@@ -38,7 +36,7 @@ public void invoke(String... parameters) {
             JRubyAsciidoctor asciidoctor = buildAsciidoctorJInstance(asciidoctorCliOptions);
             
             if (asciidoctorCliOptions.isVersion()) {
-                System.out.println("AsciidoctorJ " + asciidoctor.asciidoctorVersion() + " [http://asciidoctor.org]");
+                System.out.println("AsciidoctorJRuby " + asciidoctor.asciidoctorVersion() + " [http://asciidoctor.org]");
```

Comment:
This comes from the find-replace, but opens an interesting question: Should we actually rename the command line tool as well to `asciidoctorjruby`?

Reply:
Indeed it's a good question but I think we should stick with `asciidoctorj` at least for now.